pkg/file/tarutil.go

```diff
@@ -40,19 +43,28 @@ func (e *ErrFileNotFound) Error() string {
 // TarIterator is a function that reads across a tar and invokes a visitor function for each entry discovered. The iterator
 // stops when there are no more entries to read, if there is an error in the underlying reader or visitor function,
 // or if the visitor function returns a ErrTarStopIteration sentinel error.
-func TarIterator(reader io.Reader, visitor TarVisitor) error {
+func TarIterator(reader io.Reader, visitor TarFileVisitor) error {
```

Comment:
I know this isn't a change in this PR, but here's a comment nonetheless 😄  — it'd be nice to improve the name `TarIterator`, specifically in two ways: 1) It's usually easier to read functions in an implementation when there's a verb-like phrase instead of a noun, and this would be an opportunity to explain what the function is doing; and 2) I'm used to "iterators" being objects or functions that produce values iteratively, as opposed to the entity that's asking for new values, so the name sets an expectation for me that's not met in the function's implementation.